contracts/IdentityManager.sol

```diff
@@ -169,13 +169,19 @@ contract IdentityManager {
       delete migrationNewAddress[identity];
       identity.transfer(newIdManager);
       MigrationFinalized(identity, newIdManager, msg.sender);
+    } else {
```

Comment:
Isn't it better to do a negative tests in the if statement like we do in most other methods?